movai_core_shared/core/message_client.py

```diff
@@ -144,11 +140,11 @@ def foraward_request(self, request_msg: dict) -> dict:
         """
         if "request" not in request_msg:
             request = {"request": request_msg}
-        self._zmq_client.send(request)
+        self._zmq_client.send(request, True)
         response_required = request_msg.get("response_required")
 
         if response_required:
-            response = self._zmq_client.recieve()
+            response = self._zmq_client.recieve(True)
```

Comment:
please add what you add True
meaning `(recieving=True)`
to make it more readable